libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -38,6 +38,10 @@
 #include <liblangutil/SourceReferenceFormatter.h>
 
 #include <boost/algorithm/string/predicate.hpp>
+#include <boost/range/adaptor/reversed.hpp>
+#include <boost/fusion/algorithm/iteration/reverse_fold.hpp>
+#include <boost/fusion/include/reverse_fold.hpp>
```

Comment:
Is this and the one above actually used or are they left-over? ``boost::fusion`` is probably overkill in most cases and I don't see it used in the code...

Reply:
i initially thought about using `reverse_fold` from boost, but wasn't quite successful. Since it is earning us not that much, I think it is safe to stick with the good old range based for loop + reverse() helper.

Reply:
Ok, so the includes should be removed :-).